internal/app/go-filecoin/node/node_test.go

```diff
@@ -233,3 +235,13 @@ func TestNodeConfig(t *testing.T) {
 		Address: "/ip4/127.0.0.1/tcp/0",
 	}, cfg.Swarm)
 }
+
+func TestGenerateTestComms(t *testing.T) {
```

Comment:
This is how I got cids for setup.json.  Suggestions for where to put this?  I'm thinking gengen binary maybe?

Reply:
sounds good

Reply:
moved to a tiny binary in gengen package